include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.h

```diff
@@ -111,11 +111,6 @@ class InnerSymbolTable : public TraitBase<ConcreteType, InnerSymbolTable> {
         ConcreteType::template hasTrait<::mlir::SymbolOpInterface::Trait>(),
         "expected operation to define a Symbol");
 
-    if (op->getNumRegions() != 1)
-      return op->emitError("expected operation to have a single region");
-    if (!op->getRegion(0).hasOneBlock())
-      return op->emitError("expected operation to have a single block");
-
```

Comment:
This is done to support making operations that have no regions/blocks IST's, like the rest of FModuleLike's (FExtModuleOp, FMemModuleOp) since we need to resolve (inner) symbols on them (ports).  Commit with just this change: https://github.com/llvm/circt/pull/3575/commits/ca13f27ba015a9707f01530cca516711c5f9c9f3 .